### PR TITLE
Support legacy notebook launcher

### PR DESCRIPTION
**Pull request by fcollonval**
Extracted from #44

### CODE REVIEW

File: jupyterlab_pullrequests/__init__.py [hunk 1]
```diff
@@ -19,6 +19,12 @@ def _load_jupyter_server_extension(server_app):
     from .base import PRConfig
     from .handlers import setup_handlers
 
+    log = server_app.log
     config = PRConfig(config=server_app.config)
-    setup_handlers(server_app.web_app, config)
+    setup_handlers(server_app, config)
```
Comment (bollwyvl) [inline jupyterlab_pullrequests/__init__.py:24]: i don't think this is ready for that yet!

File: jupyterlab_pullrequests/__init__.py [hunk 2]
```diff
@@ -19,6 +19,12 @@ def _load_jupyter_server_extension(server_app):
...
     server_app.log.info("Registered jupyterlab_pullrequests extension")
+
+
+# for legacy launching with notebok (e.g. Binder)
+_jupyter_server_extension_paths = _jupyter_server_extension_points
+load_jupyter_server_extension = _load_jupyter_server_extension
```
Comment (bollwyvl) [inline jupyterlab_pullrequests/__init__.py:30]: good start, there's also the json files that need to be deployed? or maybe we already do that...

Comment (fcollonval) [reply jupyterlab_pullrequests/__init__.py:29]: The JSON files were already well set. But this backport was missing

**Review by fcollonval — COMMENTED** [review]
Thank you for fixing this one @bollwyvl

**Comment by bollwyvl** [comment]
Cool thanks for cleaning up some of the unbundling. When next I get a
chance I'll merge into #44 and see what's still hanging out, maybe it can
actually land with the anonymous provider... Someday!

**Pull request merged**